frontend/src/scenes/batch_exports/BatchExportBackfillModal.tsx

```diff
@@ -0,0 +1,74 @@
+import { useActions, useValues } from 'kea'
+
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+
+import { LemonModal } from 'lib/lemon-ui/LemonModal'
+import { Form } from 'kea-forms'
+import { Field } from 'lib/forms/Field'
+import { batchExportLogic } from './batchExportLogic'
+import { LemonCalendarSelectInput } from 'lib/lemon-ui/LemonCalendar/LemonCalendarSelect'
+
+export function BatchExportBackfillModal(): JSX.Element {
+    const { batchExportConfig, isBackfillModalOpen, isBackfillFormSubmitting } = useValues(batchExportLogic)
+    const { closeBackfillModal } = useActions(batchExportLogic)
+
+    return (
+        <LemonModal
+            title="Create historic export"
+            onClose={closeBackfillModal}
+            isOpen={isBackfillModalOpen}
+            width={'30rem'}
+            footer={
+                <>
+                    <LemonButton
+                        type="secondary"
+                        data-attr="batch-export-backfill-cancel"
+                        disabledReason={isBackfillFormSubmitting ? 'Please wait...' : undefined}
+                        onClick={closeBackfillModal}
+                    >
+                        Cancel
+                    </LemonButton>
+                    <LemonButton
+                        form="batch-export-backfill-form"
+                        htmlType="submit"
+                        type="primary"
+                        data-attr="batch-export-backfill-submit"
+                        loading={isBackfillFormSubmitting}
+                    >
+                        Schedule runs
+                    </LemonButton>
+                </>
+            }
+        >
+            <p>
+                Triggering a historic export will create multiple runs, one after another for the range specified below.
+                The runs will export data in <b>{batchExportConfig?.interval}</b> intervals, from the start date until
+                the end date is reached.
+            </p>
+
+            <Form
+                logic={batchExportLogic}
+                formKey="backfillForm"
+                id="batch-export-backfill-form"
+                enableFormOnSubmit
+                className="space-y-2"
+            >
+                <Field name="start_at" label="Start Date" className="flex-1">
+                    {({ value, onChange }) => (
+                        <LemonCalendarSelectInput value={value} onChange={onChange} placeholder="Select start date" />
+                    )}
+                </Field>
+
+                <Field name="end_at" label="End date" className="flex-1">
+                    {({ value, onChange }) => (
+                        <LemonCalendarSelectInput
+                            value={value}
+                            onChange={onChange}
+                            placeholder="Select end date (optional)"
+                        />
+                    )}
+                </Field>
```

Comment:
Question: If backfilling an hourly export, users may also want to specify the time component of `end_at`. Would that be easy to add to the Calendar select? If not, we can also tackle this later.

Reply:
I don't think we have a calendar+time component but definitely doable in a follow up PR

Reply:
Cool, yeah, I can follow-up from this PR and add a time selector of sorts.